process/templates/WG_NAME_sandbox_stage.md

```diff
@@ -14,9 +14,13 @@ When contributing to OpenSSF any existing material for the new WG to work on, th
   
 ### Working Group References
```

Comment:
Actually I missed something. Per #262, I thought we had made having a TAC sponsor a requirement for WGs at the sandbox stage: https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements

Reply:
Agreed...this was helpful for the AIML WG.